extensions/security/spi/src/main/java/io/quarkus/security/spi/AdditionalSecuredMethodsBuildItem.java

```diff
@@ -12,7 +12,11 @@
 /**
  * Contains methods that need to have {@link jakarta.annotation.security.DenyAll} or
  * {@link jakarta.annotation.security.RolesAllowed}.
+ *
+ * @deprecated as surplus to requirements; if this build item is of interest, please open issue in the Quarkus
```

Comment:
This build item may be used, https://github.com/quarkusio/quarkus/discussions/38256

Reply:
Done, I'll resolve this comment because I respect your preference.